scaffold/tugboat/config.yml.twig

```diff
@@ -24,9 +24,8 @@ services:
       init: ./.tugboat/steps/1-init.sh
       update: ./.tugboat/steps/2-update.sh
       build: ./.tugboat/steps/3-build.sh
-{% if online_command|length > 0 %}
       online: ./.tugboat/steps/4-online.sh
-{% endif %}
+
```

Comment:
This might be the additional extra line that is breaking
https://github.com/Lullabot/drainpipe/actions/runs/14646924031/job/41103365982?pr=829#step:7:23